internal/cmd/tui.go

```diff
@@ -339,6 +341,41 @@ func (m tuiModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 
 			return m, tea.Quit
+
+		case "ctrl+w", "w":
```

Comment:
I would personally prefer the search to run the command, rather than acting as a filter. I feel this is more in line with intuition, as it is awkward to select a specific command just to have to re-select in in the tui window. cc @sourishkrout 

I am happy to have two commands, one for "find," and one for "filter," the latter of which filters and the former just executes the command immediately.